doc/releases/changelog-dev.md

```diff
@@ -252,6 +252,9 @@
 
 <h3>Documentation 📝</h3>
 
+* Updated links to PennyLane.ai in the documentation to use the latest URL format, which excludes the `.html` prefix.
+  [(#6412)](https://github.com/PennyLaneAI/pennylane/pull/6412)
+
```

Comment:
I don't think we've updated the changelog for PRs like this in the past. Maybe check with @mudit2812 to see what the core team would prefer here.

Reply:
I initially didn't, but @mudit2812 suggested I should add it to the changelog. I'm fine with removing it since it's not really a meaningful contribution to the next PL release.

Reply:
I think generally speaking, we don't need changelog entries for changes outside the code base or docs. My rule of thumb is that if a PR's work fits nicely into a single section of the changelog, it should probably end up there 😄

Reply:
I'll keep it as it is and merge the PR then, feel free to modify the change-log in case you want to update it before the next release.